tools/benchmarks/llm_eval_harness/meta_eval/README.md

```diff
@@ -178,33 +178,33 @@ metric_list:
 ```
 Here we set the `num_fewshot` to 0 as our prompts have already been converted to 5-shots, and the model generation will only stop if the generated output tokens exceeds 1024, as stated in the [mmlu-pro eval details](https://github.com/meta-llama/llama-models/blob/main/models/llama3_1/eval_details.md#mmlu-pro). We will set the `do_sample` to false and `temperature` to 0 as stated in our `eval_config` column in the dataset. We will use metric `exact_match` for this tasks and calculate the `mean` as our task aggregated number.
 
-**NOTE**: While we tried our best to create the template files, those configs and functions are created based on public third-party library and are not exactly the same as our internal implementation, so there is a chance that the reproduced numbers are slightly different.
+**NOTE**: While we tried our best to create the template files, those configs and functions are created based on public third-party library and are not exactly the same as our internal implementation, so there is a chance that the eval numbers are slightly different.
 
 ## Results
 
-Here is the comparison between our reported numbers and the reproduced numbers in this tutorial:
+Here is the comparison between our reported numbers and the eval numbers in this tutorial:
```

Comment:
I am wondering if we want to put the results out. It's a recipe meant to be showing people how to do it. But I don't know if we want another source of ground truth numbers for our models out there.

Reply:
Sure, result section removed